pkg/framework/plugins/nodeutilization/nodeutilization.go

```diff
@@ -34,6 +34,8 @@ import (
 	"sigs.k8s.io/descheduler/pkg/utils"
 )
 
+const ResourceMetrics = v1.ResourceName("MetricResource")
```

Comment:
I would expect the value to be the same as the variable name. Any reason why is it different?